src/go/rpk/pkg/cli/cmd/container/start.go

```diff
@@ -411,19 +402,55 @@ func waitForCluster(check func() error, retries uint) error {
 	)
 }
 
-func renderClusterInfo(nodes []node) {
+func renderClusterInfo(c common.Client) ([]*common.NodeState, error) {
+	nodes, err := common.GetExistingNodes(c)
+	if err != nil {
+		return nil, err
+	}
+	if len(nodes) == 0 {
+		log.Info("No Redpanda nodes detected - use `rpk container start` or check `docker ps` if you expected nodes")
+	}
+
 	t := ui.NewRpkTable(log.StandardLogger().Out)
 	t.SetColWidth(80)
 	t.SetAutoWrapText(true)
-	t.SetHeader([]string{"Node ID", "Address"})
+	t.SetHeader([]string{"Node-ID", "Status", "Kafka-Address", "Admin-Address"})
```

Comment:
Awesome, as a follow-up we should include pandaproxy too

Reply:
Added to this PR